File: CodeTest/Java/SWEA/D1_2071.java
import java.util.Scanner;
import java.lang.Math;

public class D1_2071 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int testCase = sc.nextInt();
        double avg;
        int answer;

        for(int i=1;i<=testCase;i++) {
            avg = 0;
            
            for(int j=0;j<10;j++) {
                avg+= sc.nextInt();
            }

            answer = (int)Math.round(avg/10);

            System.out.println("#" + i + " " + answer);
        }
    }
}

File: CodeTest/Python/SWEA/D1_2071.py
testCase = int(input())

for i in range(1, testCase+1):
    num_list = list(map(int, input().split()))
    avg = 0

    for n in num_list:
        avg += n
    avg = int(round(avg/len(num_list), 0))

    print(f'#{i} {avg}')